<?xml version="1.0"?>
<launch>
    <arg name="config_file" default="$(find marker_localization)/config/default.yaml"/>
    <node respawn="true" pkg="marker_localization" type="marker_localization" name="free_markers_localization" output="screen">
        <rosparam file="$(arg config_file)" command="load"/>
    </node>        
</launch>